inexor/util/legacy_time.cpp

```diff
@@ -24,6 +24,7 @@ int time_since_program_start()
 }
 
 /// Block for a specific time to limit frames per seconds to use resources intelligently.
+/// @param max_fps the maximum frames per seconds the game can run at
```

Comment:
if you move the doc comment to the .hpp file, could you remove it from the c++ file? :)